test/controllers/home_controller_test.rb

```diff
@@ -0,0 +1,9 @@
+require 'test_helper'
+
+class HomeControllerTest < ActionController::TestCase
+  test "should get home" do
```

Comment:
Testing is a huge topic that you'll be getting into a lot more later. But for now I'll just mention that we typically don't write controller specs. Our typically test suite contains unit tests for any classes we create, and feature specs for describing all of the behavior of the site. Check out https://github.com/jnicklas/capybara#using-capybara-with-rspec to get a sense for what those feature specs can look like. You can also take a look at all of our pull requests that we post in Slack to get a sense for how each of us handles testing.